inc/saiswitch.h

```diff
@@ -1052,8 +1057,9 @@ typedef enum _sai_switch_attr_t
      * @brief The hash object for IPv6 packets going through LAG
      *
      * @type sai_object_id_t
-     * @flags READ_ONLY
+     * @flags CREATE_AND_SET
      * @objects SAI_OBJECT_TYPE_HASH
+     * @allownull true
```

Comment:
As I understand
When assigning null to ECMP/LAG specific traffic type (ipv4, ipv6, ipv4inipv4)
The policy applied for that traffic will be decided by the default ECMP/LAG hash - SAI_SWITCH_ATTR_ECMP_HASH, SAI_SWITCH_ATTR_LAG_HASH